src/python/nightlyBench.py

```diff
@@ -904,19 +904,22 @@ def run():
   index = comp.newIndex(NIGHTLY_DIR, mediumSource,
                         analyzer='StandardAnalyzerNoStopWords',
                         postingsFormat='Lucene90',
-                        numThreads=1,
-                        useCMS=False,
+                        numThreads=constants.INDEX_NUM_THREADS,
+                        useCMS=True,
+                        maxConcurrentMerges=12,
                         directory=DIR_IMPL,
                         idFieldPostingsFormat='Lucene90',
-                        mergePolicy='LogDocMergePolicy',
+                        mergePolicy='TieredMergePolicy',
                         facets = (('taxonomy:Date', 'Date'),
                                   ('taxonomy:Month', 'Month'),
                                   ('taxonomy:DayOfYear', 'DayOfYear'),
                                   ('sortedset:Month', 'Month'),
                                   ('sortedset:DayOfYear', 'DayOfYear')),
                         addDVFields=True,
                         vectorFile=constants.GLOVE_VECTOR_DOCS_FILE,
-                        vectorDimension=100,)
+                        vectorDimension=100,
+                        rearrange=555,
```

Comment:
Awesome!

I struggled to find the place in current benchmarks that effectively implements the `555` logic, and finally found it here: https://github.com/mikemccand/luceneutil/blob/master/src/python/competition.py#L140

And above we define `SEGS_PER_LEVEL = 5`, so indeed your `555` will achieve the same segment geometry (well, with different docs assigned to each segment).